libminifi/src/RemoteProcessorGroupPort.cpp

```diff
@@ -193,17 +193,17 @@ void RemoteProcessorGroupPort::onTrigger(const std::shared_ptr<core::ProcessCont
 
   std::unique_ptr<sitetosite::SiteToSiteClient> protocol_ = nullptr;
   try {
-    logger_->log_info("get protocol in on trigger");
+    logger_->log_trace("get protocol in on trigger");
     protocol_ = getNextProtocol();
 
     if (!protocol_) {
-      logger_->log_info("no protocol, yielding");
```

Comment:
This one should likely be debug otherwise we'd need to get to trace to support this. The one above is definitely a trace.